### DISCUSSION TITLE
可透視牆壁分辨目標，以色列開發 AI 手持穿牆感應雷達

### DISCUSSION THREAD

**Discussion by Yao900108** [1101-GS4719-C/Python-Community-GS4719-C#663] [科技新知]
以色列軍工廠商 Camero Tech 近日發表旗下最新雷達產品，不但輕巧可由單兵攜帶操作，且透過 AI 和 3D 感應技術，可偵測牆後生物或障礙物活動，還可由高度體積分辨目標為成人或孩童，為軍事、執法和情報單位提供重要資訊。
透過 AI 演算法及專利的「3D 穿牆感應」技術，新版雷達在維持輕便手持體積之餘，能夠快速辨別牆後物體是生物、雜物或建築物。
AI 演算法可透過生物特徵，分辨目標人物目前姿勢是站立、坐姿或倒臥，並由體積和形狀計算人物身高，可協助操作人員分辨目標大約年齡和體型，避免誤傷孩童或人質，對於救援任務來說相當重要。
從小就覺得透視的技術很酷，這些年也越來越成熟了!!
https://technews.tw/2022/06/10/israeli-company-developed-portable-rader-that-can-see-through-walls/

**Comment by Jimmy911218** [comment]
以前都以為可以穿牆看物只能魔法能做做，基本不行
想不到現在以色列已經有了這個技術。但希望能用在造福世界，不要用在打戰上

**Comment by treeann** [comment]
太誇張了八 以色列真的是戰鬥民族

**Comment by sys511613** [comment]
用這個玩躲貓貓直接無敵了

**Comment by HaoFish123** [comment]
感謝分享，雖然還是不太了解它的運作方式不過還蠻酷的。

**Comment by alexthebigpp** [comment]
外掛IRL出現了，以後每個以色列士兵都是神仙

**Comment by yenwei12138** [comment]
想當年，透視還是我最想要的超能力呢(誤

**Comment by bloodtrail0511** [comment]
感謝分享，很酷的技術

**Comment by etin2468** [comment]
感覺一定會被拿來用在戰爭ㄟ，不知道是好是壞

**Comment by 110409538** [comment]
以前東西不見,一直希望以後可以有透視把東西找出來.希望未來可以有這樣一個技術.東西不見真的很痛苦.

**Comment by orianna901001** [comment]
透視這項技術我感覺如果不好好保存，流到市面上會造成很大的男、女性恐慌，甚至會出現跟換臉技術相關的爭議。

**Comment by Jamesmoneya** [comment]
之前就有追蹤過以色列的軍事力量，他們有足夠的背景和人力可以發展軍事武器，他們的研究人員都非常的厲害，很多都是其他國家的國家研究室跑過去的，他們只會越來越進步，軍事力量在國際上大家有目共睹
